docs/stepactions.md

```diff
@@ -521,3 +522,104 @@ spec:
 ```
 
 The default resolver type can be configured by the `default-resolver-type` field in the `config-defaults` ConfigMap (`alpha` feature). See [additional-configs.md](./additional-configs.md) for details.
+
+### Controlling Step Execution with when Expressions
+
+You can define `when` in a StepAction to control its execution. 
```

Comment:
Isn't this wrong ? You can define `when` in a step (that refers a `StepAction`), but you cannot define `when` in the `StepAction` itself, can you ?

Reply:
ahh! You're right!  This should be a `step`

Reply:
updated